Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -593,3 +593,46 @@ void JitArm64::fctiwzx(UGeckoInstruction inst)
 	}
 	fpr.Unlock(V0);
 }
+
+void JitArm64::fdivx(UGeckoInstruction inst)
+{
+	INSTRUCTION_START
+	JITDISABLE(bJITFloatingPointOff);
+	FALLBACK_IF(inst.Rc);
+
+	u32 a = inst.FA, b = inst.FB, d = inst.FD;
+	fpr.BindToRegister(d, true);
```

Comment:
That's a good question, why am I just using true?
This seems like a copy and paste issue that hits every one of the floating point instructions.